CMakeLists.txt

```diff
@@ -156,6 +157,19 @@ mark_as_advanced(ENABLE_SENTRY)
 cmake_dependent_option(FORCE_LEGACY_LIBBPF "Force usage of libbpf 0.0.9 instead of the latest version." False "ENABLE_PLUGIN_LIBBPF" False)
 mark_as_advanced(FORCE_LEGACY_LIBBPF)
 
+if(ENABLE_WOLFSSL)
+        set(ENABLE_OPENSSL False)
+        # two features are incompatible
+        set(ENABLE_PLUGIN_LOGS_MANAGEMENT False)
+        set(ENABLE_H2O False)
+        # OSSL_DECODER_CTX does not exist in WolfSSL
+        set(ENABLE_ACLK False)
```

Comment:
This is likely fine for an initial implementation, but we ideally need to update the ACLK code to work with WolfSSL.

Reply:
I agree. We will need to modify our code, but I decide to proceed as a first step.